source/views/Settings.js

```diff
@@ -75,14 +75,14 @@ enyo.kind({
 			components:[
 				//Connectivity
 				{kind: "onyx.Toolbar", classes: "list-header", content: "Connectivity"},
-				{kind: "ListItem", icon: "icon.png", title: "Wi-Fi", ontap: "openPanel", targetPanel: "WiFiPanel",
+				{kind: "ListItem", icon: "assets/icons/com.palm.app.wifi/icon-256x256.png", title: "Wi-Fi", ontap: "openPanel", targetPanel: "WiFiPanel",
```

Comment:
Since we're not using the com.palm.app naming convention anymore. It might make more sense to just call it put them all in assets/icons/ and call them icon-wifi.png for example. This would apply to all.